airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -42,6 +45,11 @@ data:
         scopedImpact:
           - scopeType: stream
             impactedScopes: ["marketing_emails"]
+      2.0.0:
+        message: >-
+          This version eliminates the Property History stream in favor of creating 3 different streams, Contacts, Companies, and Deals, which can now all fetch their property history.
```

Comment:
This version replaces the `Property History` stream in favor of creating 3 different streams: `Contacts`, `Companies`, and `Deals`, which can now all fetch their property history.
          It will affect only users who use `Property History` stream, who will need to fix schema conflicts and sync `Contacts Property History` stream instead of `Property History`.

Reply:
This is an old version (2.0), customer will not see this line anymore
Current version is 4.0.0.
Git shows this lines because I updated breaking change order 4.0.0 ->

Reply:
they will see it if they havent updated yet

Reply:
updated